### DISCUSSION TITLE
Slider for Save Track Design with scenery

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16487] [Feature requests]
When saving a ride it would be nice if you could adjust the automatic selection of nearby scenery with a slider. A lot of the time the tool does a good job selecting most of the scenery, but then I have to click on each remaining piece 1 by 1. Also if there could be an option to save ride design with paths or without paths, or separate paths and scenery in the select nearby scenery tool. I don't know if this can be implemented via plugin or not

**Comment by Broxzier** [comment]
Instead of clicking each piece individually, you can hold down the mouse button and paint over the unselected scenery and paths.

**Reply by 733737** [reply]
imo the sprite selection in rct isn't easy and a tile radius selector would be preferable

**Reply by ZehMatt** [reply]
Probably something to look into after we change the track design to blueprints (eventually)

**Reply by 733737** [reply]
I remember 20 years ago when I was a kid that this irked me too
